src/main/kotlin/com/terraformation/backend/customer/api/GlobalRolesController.kt

```diff
@@ -39,7 +40,19 @@ class GlobalRolesController(
       @PathVariable("userId") userId: UserId,
       @RequestBody payload: UpdateGlobalRolesRequestPayload,
   ): SuccessResponsePayload {
-    userStore.updateGlobalRoles(userId, payload.globalRoles)
+    userStore.updateGlobalRoles(setOf(userId), payload.globalRoles)
+
+    return SimpleSuccessResponsePayload()
+  }
+
+  @ApiResponse200
+  @ApiResponse404
+  @DeleteMapping("/users/globalRoles")
+  @Operation(summary = "Remove global roles from the supplied users.")
+  fun removeGlobalRoles(
```

Comment:
Looks like all the other DELETE endpoint function names use the word "delete" rather than "remove," so maybe it'd be good to use that word here for consistency.